cfe_internal/update/update_policy.cf

```diff
@@ -15,6 +15,73 @@
 #########################################################
 
 bundle agent cfe_internal_update_policy
+# @brief This bundle is responsible for activating the policy to update inputs.
+# @description Users may define `def.mpf_update_policy_bundle` to override the
+#              default policy update bundle.
+{
+  classes:
+
+      # Define classes if we see a user is requesting a custom policy update bundle
+
+      "have_user_specified_update_bundle"
+        expression => isvariable( "def.mpf_update_policy_bundle" );
+
+      # Define classes if we are able to find the specific bundle they requested
+      # (otherwise we may get an error about undefined bundle)
+
+      "have_found_user_specified_update_bundle"
+        expression  => isgreaterthan( 0, length( "found_matching_user_specified_bundle" ) );
+
+  vars:
+      "default_policy_update_bundle" string => "cfe_internal_update_policy_cpv";
+
+      # Look for a bundle that matches what the user wants
+      "found_matching_user_specified_bundle"
+        slist => bundlesmatching( "$(def.mpf_update_policy_bundle" );
```

Comment:
This is where `)` is missing right?

Reply:
For the record, fixed here: https://github.com/cfengine/masterfiles/commit/adb0476cd62901aefb87f1e72c22cd8445b129a6